app/models/katello/content_view_version_import_history.rb

```diff
@@ -14,6 +14,12 @@ class ContentViewVersionImportHistory < Katello::Model
     validates :path, presence: true
     serialize :metadata, Hash
 
+    after_initialize do |history|
```

Comment:
Can't you just set the import_type when it's created and not use this hook?

Reply:
For AR yes.I was just trying to match -> https://github.com/Katello/katello/blob/master/app/models/katello/content_view_version_export_history.rb#L25-L27